src/puppetlabs/puppetdb/facts.clj

```diff
@@ -69,7 +69,7 @@
 (defn unescape-string
   "Strip escaped quotes from a string."
   [^String s]
-  (if (= \" (.charAt s 0))
+  (if (= \" (first s) (last s))
```

Comment:
no, I think that should behave the same as foo\"bar.

I'm not honestly sure why we're escaping quotes to begin with, and I wonder if it could be hsql related, but we can't change that behavior now.